src/appkit.cs

```diff
@@ -3390,6 +3440,7 @@ interface NSCollectionViewGridLayout
 	}
 
 	[Mac (10,11)]
+	[DisableDefaultCtor]
```

Comment:
Yes. :(

I'll have to XAMCORE_4_0 and add test exclusions. Not sure why I made this mistake in the haze of test fixing.